<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>520</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>116</green>
        <blue>118</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>116</green>
        <blue>118</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>116</green>
        <blue>118</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>110</red>
        <green>116</green>
        <blue>118</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Rotary Scanner Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="setPos_SM">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 255, 179);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(227, 255, 212);</string>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QLabel" name="modes_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Modes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="param1_label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Start Pos:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_PB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(168, 168, 84);
background-color: rgb(168, 168, 84);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_PB">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(168, 168, 84);
background-color: rgb(168, 168, 84);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect_PB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(168, 168, 84);
background-color: rgb(168, 168, 84);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>DISCONNECT</string>
    </property>
   </widget>
   <widget class="QLabel" name="devdata_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>49</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>DEVICE DATA</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>201</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(120, 167, 151);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="handleWidth">
     <number>28</number>
    </property>
    <property name="childrenCollapsible">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="singleModeRB">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Single Measuement</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rangedModeRB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Ranged Measurement</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fullModeRB">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Full Measurement</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="setEndPos_RM">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 255, 179);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(227, 255, 212);</string>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="setStartPos_RM">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 255, 179);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(227, 255, 212);</string>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="setStep_RM">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 255, 179);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(227, 255, 212);</string>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="setStep_FM">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 255, 179);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(227, 255, 212);</string>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
   </widget>
   <widget class="QLabel" name="param2_label">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>End Pos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="param3_label">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goHome_PB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(168, 168, 84);
background-color: rgb(168, 168, 84);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>GO HOME POSITION</string>
    </property>
   </widget>
   <widget class="QPushButton" name="write_to_file_PB">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(168, 168, 84);
background-color: rgb(168, 168, 84);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>WRITE DATA TO FILE</string>
    </property>
   </widget>
   <widget class="QWidget" name="device_data" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>360</width>
      <height>130</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 189, 94);</string>
    </property>
    <layout class="QFormLayout" name="deveice_data">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="leftMargin">
      <number>11</number>
     </property>
     <item row="0" column="1">
      <widget class="QSplitter" name="splitter_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLCDNumber" name="position_display">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 133, 0);
border-color: rgb(0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
       <widget class="QLCDNumber" name="distance_display">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 133, 0);
border-color: rgb(0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="actPos_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>ACTUAL POSITION:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lastMeas_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>LAST MEASUREMENT:</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="show_plot_PB">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(168, 168, 84);
background-color: rgb(168, 168, 84);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>SHOW PLOT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
